common/app/templates/javaScriptConfig.scala.js

```diff
@@ -60,7 +60,7 @@
         },
         "libs": {
             "googletag": "@{Configuration.javascript.config("googletagJsUrl")}",
-            "cmp": { "fullVendorDataUrl": "@Static("data/vendor/cmp_vendorlist.json")" },
+            "cmp": { "fullVendorDataUrl": "/commercial/cmp/vendorlist.json" },
```

Comment:
Should we use a `versioned` here to avoid issues with caching?

Reply:
I'll check how long the Fastly caching is

Reply:
Default caching is only for a few minutes.